src/components/Composer/index.js

```diff
@@ -197,13 +199,15 @@ class Composer extends React.Component {
     dragNDropListener(e) {
         let isOriginComposer = false;
         const handler = () => {
+            // Setting dropEffect for dragover is required for '+' icon on certain platforms/browsers (eg. Safari)
```

Comment:
Did you run `npm run lint`?

Reply:
I did and I'm not seeing any lint errors. Is there something wrong with this?

Reply:
All good then.